### DISCUSSION TITLE
How can I create a dropdown field populated with content of a different type?

### DISCUSSION THREAD

**Discussion by ricocsharp** [OrchardCMS/OrchardCore#12414] [General]
How can I create a dropdown field populated with content of a different type?  ex:  in the edition of company contenttype I need to use a country field wich will be a select or dropdown populated with data of the country contenttype. 

Is any examples avalilable?

thanks in advance

**Comment by hishamco** [comment]
Check `ContentPicker`

**Reply by ricocsharp** [reply]
Yes Thkns I already checked on this feature but a can not find any example of using it from code

**Reply by hishamco** [reply]
https://docs.orchardcore.net/en/latest/docs/reference/modules/ContentFields/#contentpicker-field-example
